packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SuggestionItemDefault.tsx

```diff
@@ -6,28 +6,30 @@ import { Persona, PersonaSize, IPersonaProps, PersonaPresence } from '../../../.
 import * as stylesImport from '../PeoplePicker.scss';
 const styles: any = stylesImport;
 
-export const SuggestionItemNormal: (persona: IPersonaProps) => JSX.Element = (personaProps: IPersonaProps) => {
+export const SuggestionItemNormal: (persona: IPersonaProps, suggestionProps: any) => JSX.Element = (personaProps: IPersonaProps, suggestionItemProps: any) => {
```

Comment:
make ` suggestionProps ` and ` suggestionItemProps ` optional, and they should have some type.